client/landing/stepper/declarative-flow/internals/steps-repository/goals/index.tsx

```diff
@@ -12,6 +11,7 @@ import { isGoalsBigSkyEligible } from 'calypso/landing/stepper/hooks/use-is-site
 import { ONBOARD_STORE } from 'calypso/landing/stepper/stores';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 import { getQueryArgs } from 'calypso/lib/query-args';
+import { useCreateCourseGoalFeature } from '../../hooks/use-create-course-goal-feature';
```

Comment:
@andregardi It Would be nice to use more absolute paths instead of relative ones.